cni/telemetry/service/telemetrymain.go

```diff
@@ -180,11 +171,10 @@ func main() {
 	}
 
 	if telemetry.CreateAITelemetryHandle(aiConfig, config.DisableAll, config.DisableTrace, config.DisableMetric) != nil {
-		log.Logger.Error("[Telemetry] AI Handle creation error", zap.Error(err))
+		logger.Error("[Telemetry] AI Handle creation error", zap.Error(err))
 	}
-	log.Logger.Info("[Telemetry] Report to host interval", zap.Duration("seconds", config.ReportToHostIntervalInSeconds))
+	logger.Info("[Telemetry] Report to host interval", zap.Duration("seconds", config.ReportToHostIntervalInSeconds))
```

Comment:
nit: Let's try to make these logs similar to all the others. I think we have component in the zap log so we can remove the [telemetry] from the log. Similarly for other CNI logs as well. We can create a task to address these.